### ISSUE TITLE
Emoticons, mentions via notifier?

### ISSUE THREAD

**Issue by andrewboni** [logankoester/grunt-hipchat-notifier#3]
Wondering how I can set up @mentions and emoticons to work with this plugin and the API.

[According to their docs](https://www.hipchat.com/docs/api/method/rooms/message), setting `message_format` to `text` (instead of the default `html`) should do the trick.

From the Hipchat docs:

```
text - Message is treated just like a message sent by a user. Can include @mentions, emoticons, pastes, and auto-detected URLs (Twitter, YouTube, images, etc).
```

I've specified the `message_format` within the `grunt-hipchat-notifier`, to no avail:

``` coffeescript
   hipchat_notifier:
      options:
        authToken: "..."
        roomId: "1"
        from: "Grunt"
        color: "purple"
      deploy_end:
        options:
          message: "Deploy finished to S3 (thumbsup)"
          message_format: "text"
```

Any ideas? Thanks!

**Comment by logankoester** [comment]
I think you're on the right track, maybe the config object isn't passed directly to the hipchat module so the format key you added gets ignored?

**Comment by logankoester** [comment]
I would be interested in a pull request to allow this of you get it working.

**Comment by logankoester** [comment]
@andrewboni My above assumption looks to be correct: https://github.com/logankoester/grunt-hipchat-notifier/blob/master/src/tasks/hipchat_notifier.coffee#L30

You would need to pass the message format option to this object.

**Comment by andrewboni** [comment]
Great- got it working. Will submit a pull request. Thanks!

**Comment by logankoester** [comment]
Awesome, thank you!

**Comment by logankoester** [comment]
Fixed & closed (see #4)

**Comment by logankoester** [comment]
@andrewboni Thanks again for your work. I have tested it our myself now and it is perfect.

I've also updated the hipchat-client dependency so you can use a room name value for `roomId` instead of the numeric id required by the old API.

These changes have been published to npm as `v0.2.1`. Enjoy :ship:
